Jenkinsfile

```diff
@@ -173,6 +195,12 @@ pipeline {
           sh "./scripts/ci/release-version"
         }
       }
+      post {
+        always {
+          archiveArtifacts 'pr.json'
```

Comment:
Why does this need to be archived?

Reply:
it helps debugging errors when something went wrong while updating github.